guides/10_Video_Processor.md

```diff
@@ -143,6 +143,39 @@ async process(buffers: VideoFrameBuffer[]): Promise<VideoFrameBuffer[]> {
 }
 ```
 
+More example about building a customized processor to load an external image:
```

Comment:
Can this be in a different section at the `###` level? i.e.
```
### Example: Building a resize processor
...

### Example: Building a external image overlay processor
...
```

Also this is a bit of a strange use case as is (just replacing the buffer).  I understand from linked issue that its more about CORS, but we should make that more clear.  Or make this an overlay processor which I would prefer.  If a builder really wanted to inject a static image they wouldn't bother with the processor, they would just create a MediaStream and inject directly.